plugins/pdr_deterministic_plugin/requirements.txt

```diff
@@ -0,0 +1,11 @@
+requests
```

Comment:
Did you create this file manually?

Reply:
Yes. I just looked for dependencies and added them, so the test will finish with successs.

Reply:
I will generate it automatically